<launch>
  <rosparam command="load" file="$(find kuri_person_tracking)/config/ros.yaml"/>
  <param name="person_detection/net_cfg_dir" type="string" value="$(find kuri_person_tracking)/net_cfg"/>

  <rosparam file="$(find kuri_camera)/cfg/camera_calibration_parameters.yaml" command="load"/>

  <node pkg="kuri_person_tracking" type="person_detection.py" name="bounding_box_publisher" output="screen"/>
</launch>
